tests/drivers/i2c/i2c_slave_api/prj_virtual.conf

```diff
@@ -8,5 +8,3 @@ CONFIG_I2C_EEPROM_SLAVE=y
 CONFIG_I2C_EEPROM_SLAVE_0=y
 CONFIG_I2C_EEPROM_SLAVE_1=y
 CONFIG_I2C_VIRTUAL=y
-CONFIG_I2C_EEPROM_SLAVE_0_CONTROLLER_DEV_NAME="I2C_VIRTUAL"
-CONFIG_I2C_EEPROM_SLAVE_1_CONTROLLER_DEV_NAME="I2C_VIRTUAL"
```

Comment:
This should mean that this configuration is never used during testing, because otherwise the assignment-to-undefined-variable warning (turned error) would've caught it.